focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -164,3 +189,63 @@ function DefaultPointRenderer.createPointView(x, y, rotation, iconFileTemplate,
 
   return view
 end
+
+--[[
+-- Takes a LrPhoto and returns the normalized original dimensions and cropped dimensions uninfluenced by the rotation
+-- photo - the LrPhoto to calculate the values from
+-- returns:
+-- - originalWidth, originalHeight - original dimensions in unrotated position
+-- - cropWidth, cropHeight - cropped dimensions in unrotated position
+--]]
+function DefaultPointRenderer.getNormalizedDimensions(photo)
+  local originalWidth, originalHeight = parseDimens(photo:getFormattedMetadata("dimensions"))
+  local cropWidth, cropHeight = parseDimens(photo:getFormattedMetadata("croppedDimensions"))
+  local userRotation, userMirroring = DefaultPointRenderer.getUserRotationAndMirroring(photo)
+
+  if userRotation == 90 or userRotation == -90 then
+    -- In case the image has been rotated by the user in the grid view, LR inverts width and height but does NOT change cropLeft and cropTop...
+    -- In this methods, width and height refer to the original width and height
+    local tmp = originalHeight
+    originalHeight = originalWidth
+    originalWidth = tmp
+
+    tmp = cropHeight
+    cropHeight = cropWidth
+    cropWidth = tmp
+  end
+
+  return originalWidth, originalHeight, cropWidth, cropHeight
+end
+
+--[[
+-- Takes a LrPhoto and returns the rotation and horizontal mirroring that the user has choosen  in Lightroom (generaly in grid mode)
+-- photo - the LrPhoto to calculate the values from
+-- returns:
+-- - rotation in degrees in trigonometric sense
+-- - horizontal mirroring (0 -> none, -1 -> yes)
+--]]
+function DefaultPointRenderer.getUserRotationAndMirroring(photo)
+  local userRotation = photo:getRawMetadata("orientation")
```

Comment:
Mh. I think in case getRawMetadata("orientation") returns NIL, we could simply default back to the old method of checking using the exifs. 
I'll make an issue for that, please assign it to me and I'll take care of it